ml-algorithms/src/main/java/org/opensearch/ml/engine/MLEngine.java

```diff
@@ -31,10 +33,22 @@
  */
 @Log4j2
 public class MLEngine {
+    public static final Setting<String> ML_COMMON_MODEL_REPO_ENDPOINT = Setting
```

Comment:
This setting doesn't have `Setting.Property.Dynamic` , so user can not update it when cluster started, right? Suggest add some comments to tell how to configure this and explain the limitation and why.